packages/contracts-bedrock/src/L2/CrossL2Inbox.sol

```diff
@@ -71,7 +81,7 @@ contract CrossL2Inbox is ICrossL2Inbox, ISemver, TransientReentrancyAware {
 
     /// @notice Semantic version.
     /// @custom:semver 1.0.0-beta.5
-    string public constant version = "1.0.0-beta.5";
+    string public constant version = "1.1.0-beta.5";
```

Comment:
I think its better to bump the beta version given this is unreleased, it the suffix `5` -> `6`